CONTRIBUTING.md

```diff
@@ -175,10 +175,67 @@ the page to activate debug mode. The state of debug mode will be remembered by
 `window.sessionStorage` and will not survive a browser restart or opening the
 docs in a new tab.
 
-### Invisible index anchors
-
-You can create index entries and invisible anchors with `X<|thing,category>`.
-To make them visible activate debug mode.
+### Search indexing rules
+
+Bits of the content can be indexed for quick navigation.
+
+An index item is created using the Pod formatting code `X` in those formats:
+
+    # the `text` is rendered, an item with text `item` under the category `category`
+    # is added to the index
+    X<text|category,item>
+    # can index more than one item
+    X<text|category1,item1;category2,item2;...>
+    # can omit text, nothing is rendered, but an index item will be created
+    X<|category,item>
+
+Valid examples are:
+
+    X<|Syntax,does>
+    X<|Language,\ (container binding)>
+    X<Subroutines|Syntax,sub>
+    X<|Variables,$*PID>
+    X<Automatic signatures|Variables,@_;Variables,%_>
+    X<Typing|Language,typed array;Syntax,[ ] (typed array)>
+    X<Attributes|Language,Attribute;Other languages,Property;Other languages,Member;Other languages,Slot>
+
+To avoid cluttering of index item categories, only a number of categories is allowed to specify,
```

Comment:
suggested edit: "only a number of categories can be specified,"

Reply:
Additional edit: say precisely what's that number.